generate-adaptors/index.js

```diff
@@ -131,6 +131,16 @@ const sampleConfiguration = json => {
       return conf;
     });
   }
+  if (properties && required) {
+    required.forEach((key, index) => {
+      conf[key] = 'Someting';
```

Comment:
Can we remove this line?

Reply:
✅ i have replaced that line with what you suggested 
```
conf[key] = properties[key].examples?.[0]
```